### PR TITLE
Changed README.rst

### PR DESCRIPTION
**Pull request by aqdasrehman**

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @aqdasrehman! I've created [OSPR-1316](https://openedx.atlassian.net/browse/OSPR-1316) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)     or indicated your institutional affiliation.     If you like, you can add yourself to the [AUTHORS](https://github.com/edx/edx-e2e-tests/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by aqdasrehman** [comment]
@benpatterson @raeeschachar please review

**Comment by nedbat** [comment]
I've deleted the mistaken OSPR JIRA issue.

**Comment by aqdasrehman** [comment]
@nedbat What do I have to do next? Should I create the PR again?

**Comment by nedbat** [comment]
@aqdasrehman no, this PR is fine.  @benpatterson and/or @raeeschachar can help get this finished. I'll also delete the bot's comment above.

File: README.rst [hunk 1]
```diff
@@ -9,8 +9,7 @@ Overview
 UI-level tests for edX applications:
 
 - ``pages``: PageObjects for interacting with pages under test.
-- ``test_lms``: Selenium tests for the Learning Management System (LMS).
-- ``test_studio``: Selenium tests for Studio.
+- ``e2e_test``: Selenium tests for the Learning Management System (LMS) and Studio.
```
Comment (raeeschachar) [inline README.rst]: Bokchoy tests for the Learning Management System (LMS) and Studio.

File: README.rst [hunk 2]
```diff
@@ -36,58 +35,65 @@ See `edx/configuration <http://github.com/edx/configuration>`_ for instructions
...
+    paver install_pages
 
-Edit the configuration file ``config.ini`` to provide information about the system under test.
-You can specify another configuration file by setting the ``CONFIG_PATH`` environment variable.
 
 
 Running Tests Locally
 ---------------------
 
 Within the Vagrant environment, the tests are installed in /opt/dev/edx-e2e-tests,
```
Comment (raeeschachar) [inline README.rst]: Did you mean -> /edx/src/edx-e2e-tests$ ?

File: README.rst [hunk 3]
```diff
@@ -36,58 +35,65 @@ See `edx/configuration <http://github.com/edx/configuration>`_ for instructions
 
 Configuration
 -------------
+1. Install requirements:
+
+.. code:: bash
+
+    pip install -r requirements/base.txt
```
Comment (raeeschachar) [inline README.rst]: Should we tell where to run this command?

File: README.rst [hunk 4]
```diff
@@ -36,58 +35,65 @@ See `edx/configuration <http://github.com/edx/configuration>`_ for instructions
...
+
+2 - Install edx platform pages:
+
+.. code:: bash
+
+    paver install_pages
```
Comment (raeeschachar) [inline README.rst]: and this too?

File: README.rst [hunk 5]
```diff
@@ -36,58 +35,65 @@ See `edx/configuration <http://github.com/edx/configuration>`_ for instructions
...
-    fab test_studio
 
 
 The commands also accept nose-style specifiers for test case or module:
 
 .. code:: bash
 
-    fab test_lms:test_lms.py:RegistrationTest.test_register
-    fab test_studio:test_studio.py:LoggedOutTest
+    paver e2e_test lms/test_dasboard.py: DashboardTest.test_resume_course
```
Comment (raeeschachar) [inline README.rst]: To run all the tests in the file:
paver e2e_test lms/test_dasboard.py

To run all the tests in a particular class:
paver e2e_test lms/test_dasboard.py: DashboardTest

To run a single test:
paver e2e_test lms/test_dasboard.py: DashboardTest.test_resume_course

**Comment by raeeschachar** [comment]
@aqdasrehman 
Reviewed. Please address comments.

File: README.rst (hunk 2 above)
Comment (Muddasser) [inline README.rst]: @raeeschachar There are two possible methods of configuring the repo. One within an existing vagrant machine e.g edx-platform devstack and other within a standalone vagrant machine for edx-e2e-tests. The documentation is about configuring the repo within a standalone vagrant machine. What are you suggesting of doing?

Comment (raeeschachar) [inline README.rst]: @Muddasser 
This should work than.

**Comment by Muddasser** [comment]
👍

**Comment by raeeschachar** [comment]
👍 merge it.

**Pull request merged**